Jenkinsfile

```diff
@@ -313,6 +313,41 @@ def buildHipClangJobAndReboot(Map conf=[:]){
     }
 }
 
+def RunPerfTest(Map conf=[:]){
+    def dockerOpts="--device=/dev/kfd --device=/dev/dri --group-add video --group-add render --cap-add=SYS_PTRACE --security-opt seccomp=unconfined"
+    try {
+        (retimage, image) = getDockerImage(conf)
+        withDockerContainer(image: image, args: dockerOpts + ' -v=/var/jenkins/:/var/jenkins') {
+        timeout(time: 600, unit: 'MINUTES')
+            {
+                unstash 'miopen_tar'
+                sh "tar -zxvf build/miopen-hip-*-Linux-runtime.tar.gz"
+                ld_lib="${env.WORKSPACE}/opt/rocm/lib"
+                def filename = conf.get("filename", "Resnet50_v1.5.txt") // default Resnet50_v1.5 
```

Comment:
[Recommendation] Remove the default to avoid false positives (unlikely but still possible)

Reply:
Resolved.